docs/pages/api-reference/liveblocks-react-comments.mdx

```diff
@@ -214,18 +236,34 @@ Displays a thread of comments, with a composer to reply to it.
 Displays a single comment.
 
 ```tsx
-<>
-  {thread.comments.map((comment) => (
-    <Comment key={comment.id} comment={comment} />
-  ))}
-</>
+<Comment comment={comment} />
 ```
 
+Combine with [`useThreads`](/docs/api-reference/liveblocks-react#useThreads) to
```

Comment:
I don't know if we want to phrase it like that, they don't have to be strictly combined, like in this example where `thread` isn't directly coming from `useThreads` but as a prop. I think I see "combine" as too strong.